deployment/terraform/create.go

```diff
@@ -181,8 +220,34 @@ func (t *Terraform) setupAppServers(output *terraformOutput, extAgent *ssh.ExtAg
 	}
 }
 
+func (t *Terraform) setupLoadtestAgents(extAgent *ssh.ExtAgent, output *terraformOutput) error {
+	// Creating the sysadmin user.
+	// should wait the server to start
+
+	for i := 0; i < len(output.Agents.Value); i++ {
+		val := output.Agents.Value[i]
+
+		if err := t.runAgent(extAgent, val.PublicIP); err != nil {
+			mlog.Error("error while setting up an agent", mlog.String("agent name", val.Tags.Name), mlog.Err(err))
+			continue
+		}
+	}
+
+	if err := t.updateCoordinatorConfig(extAgent, output); err != nil {
+		mlog.Error(err.Error())
+		// return err
+	}
+
+	if err := t.startCoordinator(extAgent, output.Agents.Value[0].PublicIP); err != nil {
```

Comment:
I am still thinking whether we want to start the coordinator immediately with the deployment. This means a deployment will essentially start running the load test.

I was thinking to have a separate command for it like "./cmd/deployer run", which would get the *terraformOutput again and just call this method.

Logically, running the load test is a separate step from deployment IMO. If the operator wants to run them both, then they can just `&&` the commands.

Not strongly opinionated on this though. Any thoughts ?

Reply:
In my opinion, since we are setting everything for loadtesting, it *should* start straight. But if we need to do something before running the coordinator, it would make sense. What could be the reason not to start load test?

Reply:
> What could be the reason not to start load test?

Actually, thinking more about it. We would probably reuse the same environment to run multiple load tests with different versions of mattermost. In that case, after running a test, we would upload a different binary and run the test again. So we might need a separate command to stop/start the load test. 

Hence, I was thinking that deployment just deploys. And then we have a separate command that starts/stop the coordinator.

Reply:
Sounds reasonable. 👍